src/statistics.jl

```diff
@@ -1492,7 +1492,7 @@ const contour = ContourStatistic
 function default_scales(::ContourStatistic)
     return [Gadfly.Scale.z_func(), Gadfly.Scale.x_continuous(),
             Gadfly.Scale.y_continuous(),
-            Gadfly.Scale.color_continuous_gradient()]
+            Gadfly.get_scale(:numerical, :color)]
```

Comment:
I'll leave the others unchanged for now, because its seems this will in fact be faster than a dict lookup. We should consider this if we are doing a larger simplification of these things.